### ISSUE TITLE
Singular unit of 1.5 (should be plural?)

### ISSUE THREAD

**Issue by eerimoq** [xolox/python-humanfriendly#43]
Hello, and thanks for a great package!

The script below outputs ``1.5 second``. Shouldn't it output ``1.5 seconds``?

``` python
import humanfriendly

print(humanfriendly.format_timespan(1.5))
```

See https://ell.stackexchange.com/questions/7817/singular-or-plural-for-seconds.

**Comment by xolox** [comment]
Hi Erik, thanks for the feedback and sorry for the late response.

I've just published [humanfriendly 9.0](https://humanfriendly.readthedocs.io/en/latest/changelog.html#release-9-0-2020-12-01) which fixes the issue you reported.

Cheers!

**Comment by eerimoq** [comment]
Thanks!
